_data/simple-icons.json

```
@@ -3398,6 +3398,16 @@
             "hex": "BE2323",
             "source": "https://www.ebox.ca"
         },
+        {
+            "title": "Eclipse Adoptium",
+            "hex": "FF1464",
+            "source": "https://adoptium.net",
+            "guidelines": "https://www.eclipse.org/org/artwork/guidelines/adoptium-brand-guidelines.pdf",
+            "license": {
+                "type": "EPL-2.0",
+                "url": "https://www.eclipse.org/legal/epl-2.0/"
```

```suggestion
                "url": "https://www.eclipse.org/legal/epl-2.0"
```
Trailing slash unnecessary